### PR TITLE
Add some topics to introduction

### PR DESCRIPTION
**Pull request by flotob**
Added some ideas/topics. Nothing much :)

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
 
 * Are computers and robots morally responsible for their actions? 
 
+### Autonomous Machines
+
+* Can software possess autonomy?
```
Comment (arielelkin) [inline introduction.md:55]: "autonomy" is ambiguous. Do you mean:
- a computer's _right_ to control itself? 
- whether a computer can manage itself without the assistance or intervention of a human? 
- Kantian autonomy, i.e. 

> the capacity of an agent to act in accordance with objective morality rather than under the influence of desires.

File: introduction.md [hunk 2]
```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
...
+
+* Is autonomy a subjective or objective quality? 
+
+* Is there a need to change our definition of autonomy?
+
+### Memetics: replication and curation
+
+* How can the intellectual framework of memetics be applied to understand modern information technology?
```
Comment (arielelkin) [inline introduction.md:63]: Memetics' fundamental premise is that: 

> Cultural evolution, including the evolution of knowledge, can be modelled through the same basic principles of variation and selection that underly biological evolution. 

`-` [Principia Cybernetica](http://pespmc1.vub.ac.be/memes.html)

Memetics is suspicious. There is an empirical basis for biological selection: genes can be singled out and observed, changes in a population's genotype have been observed. What is the empirical basis for memetics? A meme can be _any_ information pattern, so there is no way to reliably distinguish two similar memes. 

And even if we assume there is such a way, there is no way to observe the meme replicating itself _inside people's heads_. A computer can tell you that a YouTube video has been shared a million times, but cannot tell you whether or not, or how, the video is stored in people's minds.

The intellectual framework of memetics _is_ the analogy with biological evolution. How can biological evolution help us understand IT? Plainly, hardware and software are getting progressively more complex, efficient, and powerful. You could talk of certain technologies being _selected_ over competing technologies. But genes and technology are fundamentally different. Genes can be selected if they're the result of random mutations. What would be the corresponding observable phenomenon in technological evolution? Genes are inherited, and lineage/parenthood can be empirically traced. Again, what's the empirical equivalent in technology? 

So I don't think evolution's intellectual framework can shed much light on IT.

File: introduction.md [hunk 3]
```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
...
+
+#### Economics and Heteromation
+
+* Is there an unexplained interdependence between technology and economics (which we are only slowly uncovering)?
```
Comment (arielelkin) [inline introduction.md:67]: Surely the two _are_ interdependent. Economic activity relies on technology: assembly lines, word processors, satellites, etc. And technologies are developed to address or create demand. 

What is left to uncover?

File: introduction.md (hunk 2 above)
Comment (flotob) [inline introduction.md:63]: Great argument. I agree that Memetics is suspicious and should therefore be left out. Let me rephrase my question: Since Luhman's "the medium is the message" it is generally acknowledged that technological evolution drives cultural evolution. However, is the reverse also true? How much does our cultural evolution feed back on our technological evolution? [to be expandend]
